src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx

```diff
@@ -310,9 +311,20 @@ export const ChooseRightsPage = () => {
         size={isSm ? 'small' : 'medium'}
       >
         <PageHeader icon={<PersonIcon />}>{t('single_rights.delegate_single_rights')}</PageHeader>
-
         <PageContent>
-          {!isLoading && recipientError ? (
+          {chosenServices.length < 1 ? (
```

Comment:
Just leaving a comment here so that you can re-request review on this after adding the changes that were discussed on Slack 🌱